config/routes.rb

```diff
@@ -10,7 +10,8 @@
   get 'users/none' => 'users#none'
 
   resources :users
-  resources :groups
+  resources :groups, controller: 'teams' # preserve old URLs
```

Comment:
Do we know if people have bookmarked the `/groups` URL? Do we care? I think there's a case to be made to not keep the old URLs at all

Reply:
Agreed we don't need to keep the old URLs